internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -96,6 +98,15 @@ COPY roles/ ${HOME}/roles/[[ end ]]
 [[- if .Playbook ]]
 COPY playbook.yml ${HOME}/playbook.yml[[ end ]]
 
+[[- if or .Roles .Playbook ]]
+# Corrects file permissions to be fully executable and writable
+RUN find ${HOME} -type f -exec 'chmod -R g+rw {}'  && \
+	find ${HOME} -type d -exec 'chmod -R g+rwx {}' 
+[[ else if .Watches ]]
+# Corrects file permissions to be fully executable and writable
+RUN find ${HOME} -type f -exec 'chmod -R g+rw {}'  && \
+    find ${HOME} -type d -exec 'chmod -R g+rwx {}' [[ end ]]
```

Comment:
Nit: remove the space between `'` and `[[ end ]]`. This would result in the line ending in a space character.
```suggestion
    find ${HOME} -type d -exec 'chmod -R g+rwx {}'[[ end ]]
```
Ditto for the helm hybrid Dockerfile.

Reply:
Also it looks like [`find` is complaining about the `-exec` syntax](https://travis-ci.org/operator-framework/operator-sdk/jobs/574989601#L2276)

I think it needs to be:
```suggestion
    find ${HOME} -type d -exec chmod -R g+rwx {} \;[[ end ]]
```

Again, ditto for the other `find` commands in all the files.